documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
+
+## Goals
+Goals of conducted tests are as following:
+* Assess system performance,
```

Comment:
You seem to be using Polish punctuation rules.

Check out [this article](https://en.oxforddictionaries.com/punctuation/bullet-points) for an overview of punctuation in bullet points.

TL;DR the comma looks strange. You can use periods in each bullet points or just refrain from using punctuation marks at the end of each line.